orchagent/aclorch.cpp

```diff
@@ -1257,7 +1586,7 @@ AclRange *AclRange::create(sai_acl_range_type_t type, int min, int max)
 
         // work around to avoid syncd termination on SAI error due to max count of ranges reached
         // can be removed when syncd start passing errors to the SAI callers
-        char *platform = getenv("platform");
+        char *platform = getenv("onie_platform");
```

Comment:
this needs to be kept as platform, we need to support non onie-base platform.

Reply:
Will fix it